DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -86,22 +85,23 @@ struct Triggers {
 };
 
 struct Digit {
-  o2::dataformats::RangeRefComp<5> ref;
-
+  o2::dataformats::RangeReference<uint32_t, uint32_t> ref;
```

Comment:
The CI fails due to the narrowing (when using this ref.) from uint to int. 
If you indeed need to use full RangeReference, to avoid conversion, you better use ``o2::dataformats::RangeReference<int, int>``. 

Note, that later this range is assigned to RecPoints, which is still using ``RangeRefComp<5>``  : https://github.com/AliceO2Group/AliceO2/blob/11780799d4605486dfb357b44136d823b8028fb9/DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RecPoints.h#L60
I guess they should be consistent? So, what is in the end the max N of channels per digits (or recpoint) ?

Reply:
Yes, this should be fix.